### DISCUSSION TITLE
visibility layers

### DISCUSSION THREAD

**Discussion by AndHalfAFish** [godotengine/godot-proposals#4814] [Editor]
It would save time if we could assign visibility layers to entire scenes, instead of only to the individual meshes in the scene

![visibility_layers](https://user-images.githubusercontent.com/59080934/177251907-2b71ffdb-91a2-4358-bff9-5314b8094dc7.png)
.

**Comment by Calinou** [comment]
Can't you do this using a script or multiple node selection?

**Comment by AndHalfAFish** [comment]
Yes, you could do it that way. But if you have many scenes, it starts to feel like busy-work.

I think in most use-cases, if one wants an object to be in a different visibility layer, one would want all meshes of that object in the different layer ; not just one. I feel like this could be a nice usability improvement.

**Comment by azur-wolve** [comment]
Maybe you can do it using a tool script, not sure how rn though.
